diff --git a/node_modules/react-native-camera-kit/android/src/main/java/com/rncamerakit/RNCameraKitModule.kt b/node_modules/react-native-camera-kit/android/src/main/java/com/rncamerakit/RNCameraKitModule.kt
index be8ad9a..a8b0f32 100644
--- a/node_modules/react-native-camera-kit/android/src/main/java/com/rncamerakit/RNCameraKitModule.kt
+++ b/node_modules/react-native-camera-kit/android/src/main/java/com/rncamerakit/RNCameraKitModule.kt
@@ -34,7 +34,9 @@ class RNCameraKitModule(private val reactContext: ReactApplicationContext) : Rea
         val uiManager = context.getNativeModule(UIManagerModule::class.java)
         context.runOnUiQueueThread {
             val view = uiManager?.resolveView(viewTag) as CKCamera
-            view.capture(options.toHashMap(), promise)
+            @Suppress("UNCHECKED_CAST")
+            val captureOptions = options.toHashMap() as Map<String, Any>
+            view.capture(captureOptions, promise)
         }
     }
 }
\ No newline at end of file
